packages/react/src/SubdomainNavBar/SubdomainNavBar.tsx

```diff
@@ -322,7 +322,6 @@ function Root({
                     )}
                     {hasLinks && !menuHidden && (
                       <NavigationVisbilityObserver
-                        showOnlyOnNarrow
```

Comment:
Oh wow, so this wasn't doing anything at all? 😬

Reply:
No I don't believe so 😅